src/containers/Game/BattleArena.js

```diff
@@ -6,29 +6,77 @@ import HealthBar from '../../components/Game/HealthBar'
+const mageattack = require ('../../img/classes/mageattack.gif')
+
 export default class BattleArena extends PureComponent {
   static propTypes = {
     background: PropTypes.string.isRequired,
     player1: PropTypes.string.isRequired,
     player2: PropTypes.string.isRequired,
   }
   state = {
-    player1status :'move',
-    player2status :'move'
+    playerStatus :'idle',
   }
 
-CalulatePercentage = (totalDamage) => {
+calulatePercentage = (totalDamage) => {
   const fullHealth = 10
   return ((fullHealth-totalDamage)*(fullHealth))
 }
+toggleState = (stateString) => {
+  this.setState({
+    playerStatus: stateString
+  })
+}
 
   render() {
     return (
       <div className={this.props.background}>
-      <HealthBar side='left' percentage={this.CalulatePercentage(0)}/>
-      <Player side='left' imgSrc={this.props.player1} status={this.state.player1status}/>
-      <HealthBar side='right' percentage={this.CalulatePercentage(0)}/>
-      <Player side='right' imgSrc={this.props.player2} status={this.state.player2status}/>
+        {this.state.playerStatus === 'idle' &&
+        <div>
+          <HealthBar side='left' percentage={this.calulatePercentage(0)}/>
+          <Player side='left' imgSrc={fighteridle} status={this.state.playerStatus}/>
+          <HealthBar side='right' percentage={this.calulatePercentage(0)}/>
+          <Player side='right' imgSrc={mageidle} status={this.state.playerStatus}/>
+          <button onClick={_ => this.toggleState('move')}/>
+        </div>
+        }
+        {this.state.playerStatus === 'move' &&
+        <div>
+          <HealthBar side='left' percentage={this.calulatePercentage(0)}/>
+          <Player side='left' imgSrc={fightermove} status={this.state.playerStatus}/>
+          <HealthBar side='right' percentage={this.calulatePercentage(0)}/>
+          <Player side='right' imgSrc={magemove} status={this.state.playerStatus}/>
+          <button onClick={_ => this.toggleState('attack')}/>
+        </div>
+        }
+        {this.state.playerStatus === 'attack' &&
+        <div>
+          <HealthBar side='left' percentage={this.calulatePercentage(0)}/>
+          <Player side='left' imgSrc={fighterattack} status={this.state.playerStatus}/>
+          <HealthBar side='right' percentage={this.calulatePercentage(0)}/>
+          <Player side='right' imgSrc={mageattack} status={this.state.playerStatus}/>
+          <button onClick={_ => this.toggleState('moveback')}/>
+        </div>
+        }
+        {this.state.playerStatus === 'moveback' &&
+        <div>
+          <HealthBar side='left' percentage={this.calulatePercentage(2)}/>
+          <Player side='left' imgSrc={fightermove} status={this.state.playerStatus}/>
+          <HealthBar side='right' percentage={this.calulatePercentage(0)}/>
+          <Player side='right' imgSrc={magemove} status={this.state.playerStatus}/>
+          <button onClick={_ => this.toggleState('idle')}/>
+        </div>
+        }
```

Comment:
Nice conditions.

Isn't it possible for healthbar to be outside the condition though since it doesn't move?